README.md

```diff
@@ -49,7 +49,7 @@ The site should now be accessible on `localhost:4000` on the host, and can be mo
 
 ### Building for Deployment
```

Comment:
Hi @kai33, thanks for the feedback, I think there's a little confusion. If the dependency is not local, the website will continue to use a link to download the dependency. Only when the dependency are local, will it concatenate to the respective html files. The user can put the local js/css files anywhere they want, the rake task will find it and append :)